config/locales/en/main.yml

```diff
@@ -2058,6 +2058,9 @@ en:
     change_twilio_auth_token: "Change Auth Token"
     change_api_key: "Change API Key"
     override_code_warning: "Note that the new override code will not work with forms that have already been downloaded. You should record the existing code if this is a problem. Are you sure you want to regenerate the code?"
+    sms_token_warning: "Note that this will invalidate the previous SMS token for this mission. Are you sure you want to regenerate the token?"
+    api_key_warning: "Note that this will invalidate the previous API key for this user. Are you sure you want to regenerate the key?"
+    sms_auth_code_warning: "Note that this will invalidate the previous SMS auth code for this user. Are you sure you want to regenerate the code?"
```

Comment:
I went with "for this user" to make it generic for yourself vs. a user you're editing.

Let me know if you want more detail e.g. describe what the actual consequences are like "you need to change the code you use when sending future SMS messages" but that seems hopefully clear from context.

Reply:
👍